src/main/java/uk/ac/sanger/sccp/stan/service/EmailService.java

```diff
@@ -70,4 +77,33 @@ public boolean tryAndSendAlert(String subject, String text) {
     public String getServiceDescription() {
         return mailConfig.getServiceDescription();
     }
+
+    /**
+     * Tries to send a release email.
+     * @param recipient the recipient of the release email
+     * @param releaseFilePath the path to download the release file
+     * @return true if the email was sent successfully; false if it was not
+     */
+    public boolean tryReleaseEmail(String recipient, String releaseFilePath) {
+        String[] recipients = new String[] {recipient};
+        String releaseCc = mailConfig.getReleaseCC();
+        String[] cc;
+        if (nullOrEmpty(releaseCc)) {
+            cc = null;
+        } else {
+            if (releaseCc.indexOf('@') < 0) {
+                releaseCc += "@sanger.ac.uk";
+            }
+            cc = new String[] { releaseCc };
+        }
+        String subject = mailConfig.getServiceDescription()+" release";
+        String text = "The details of the release are available at "+releaseFilePath;
+        try {
+            send(subject, text, recipients, cc);
+            return true;
+        } catch (Exception e) {
+            log.error("Failed to send release email.", e);
+            return false;
+        }
+    }
 }
```

Comment:
Just to understand- Are there two set of recipients? 
1) Those who need to alert (tryAndSendAlert) and 
2) general mail id to notify release operation  (tryReleaseEmail)
Are these both groups are notified together?

Reply:
If there is an *alert* (something has gone wrong in the application) the developers will be notified.
The *release email* is sent to the listed recipient for that release, with cc to sgp@sanger.ac.uk as specified in the story.